ansible/roles/ocp4-workload-integreatly/tasks/workload.yml

```diff
@@ -33,6 +33,72 @@
   - "Variable 2:      {{ ocp4_workload_integreatly.variable_2 }}"
   - "Variable Secret: {{ ocp4_workload_integreatly.variable_secret }}"
 
+- name: Get cluster token
```

Comment:
@ciaranRoche Wondering should this logic be in the post_workload task instead? 

In my head at a very high level the process would be as follows:
**pre_workload**: _Install and configure Minio_
**workload**: _Install integreatly-operator and an instance of RHMI. Wait for install to complete_
**post_workload**: _Setup users, create user specific resources for fuse/3scale etc._

Happy to hear your thoughts on the above though ^^ 

Pinging @aidenkeating @austincunningham @dimitraz for visibility

Reply:
Yeah I was not 100% sure where this was to go, I initially was leaning towards `post_workload`. Happy to move.